qiskit/circuit/qpy_serialization.py

```diff
@@ -160,6 +160,90 @@
 which is immediately followed by ``pauli_op_size`` bytes which are .npy format [#f2]_
 data which represents the :class:`~qiskit.quantum_info.SparsePauliOp`.
 
+Version 3 of the QPY format also defines a struct format to represent a
+:class:`~qiskit.circuit.ParameterVectorElement` as a distinct subclass from
+a :class:`~qiskit.circuit.Parameter`. This adds a new parameter type char ``'v'``
+to represent a :class:`~qiskit.circuit.ParameterVectorElement` which is now
+supported as a type string value for an INSTRUCTION_PARAM. The payload for these
+parameters are defined below as :ref:`param_vector`.
+
+.. _param_vector:
+
+
+PARAMETER_VECTOR_ELEMENT
+------------------------
+
+A PARAMETER_VECTOR_ELEMENT represents a :class:`~qiskit.circuit.ParameterVectorElement`
```

Comment:
`ParameterVectorElement` technically doesn't have any documentation generated, but that's more a bug on its behalf than here.  At some point I'd be in favour of us turning on some option in Sphinx to make it an error to write a failed cross-ref, but for now let's just leave it.